src/NuGet.Core/NuGet.PackageManagement/NuGet.PackageManagement.csproj

```diff
@@ -1,10 +1,17 @@
 ﻿<Project>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.props" />
+  <!--
```

Comment:
I would remove the comment. 
We do the same thing throughout our project right now.